src/app/orders/orderList/js/oc-orders.service.js

```diff
@@ -48,11 +48,11 @@ function ocOrdersService(OrderCloudSDK){
 
         if(parameters.tab === 'grouporders') {
             if(parameters.group) {
-                return OrderCloudSDK.Me.ListAddresses({CompanyName: parameters.group})
-                    .then(function(address) {
-                        var shippingAddressID = address.Items[0].ID;
-                        parameters.filters = {ShippingAddressID: shippingAddressID, Status: parameters.status};
-                        return OrderCloudSDK.Me.ListOrders(parameters);
+                return OrderCloudSDK.Me.ListAddresses()
+                    .then(function(addresses) {
+                        var shippingAddress = _.where(addresses.Items, {CompanyName: parameters.group});
+                        parameters.filters = {ShippingAddressID: shippingAddress[0].ID, Status: parameters.status};
```

Comment:
In the case that they may want to filter on a group _and_ other filters (other than Status), you could use:

angular.extend(parameters.filters, {ShippingAddressID: shippingAddress[0].ID});

Then parameters.filters will still have Status, DateSubbmited, etc. on it, but it will _also_ have ShippingAddressID. This is totally based on what they're looking for, but if they want to filter on group and other things as well, that would be the way to go.